Source/Xamarin/Prism.Forms/Navigation/NavigationMode.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Prism.Navigation
+{
+    public enum NavigationMode
+    {
+        Forward,
```

Comment:
Should probably rename this to "New", since technically we aren't navigating forward but instead creating a new page and pushing it onto the nav stack.  Forward gives the impression that we have an existing page we are going forward to in the nav stack